selenium.go

```diff
@@ -223,6 +225,37 @@ const (
 	SameSiteEmpty           = ""
 )
 
+type PointerType string
```

Comment:
Add commentary for all of these exported identifiers.

https://golang.org/doc/effective_go#commentary